fred2/sexp_tree.cpp

```diff
@@ -7361,9 +7369,15 @@ int sexp_tree::get_container_usage_count(const SCP_string &container_name) const
 
 bool sexp_tree::rename_container_nodes(const SCP_string &old_name, const SCP_string &new_name)
 {
-	Assert(!old_name.empty());
-	Assert(!new_name.empty());
-	Assert(new_name.length() <= sexp_container::NAME_MAX_LENGTH);
+	Assertion(!old_name.empty(),
+		"Attempt to rename container nodes looking for empty name. Please report!");
+	Assertion(!new_name.empty(),
+		"Attempt to rename container nodes with empty name. Please report!");
+	Assertion(new_name.length() <= sexp_container::NAME_MAX_LENGTH,
```

Comment:
Just to check... is `NAME_MAX_LENGTH` a length or a buffer size?

Reply:
It's a length (see sexp_container.h).